packages/blocks/src/__internal__/utils/check-line.ts

```diff
@@ -143,10 +143,11 @@ function shiftRange(range: Range): Range | null {
  */
 export function isAtLineEdge(range: Range) {
   if (!range.collapsed) {
-    console.warn(
-      'Failed to determine if the caret is at line edge! expected a collapsed range but got',
-      range
-    );
+    // FIXME
```

Comment:
What does it break?

Reply:
https://github.com/toeverything/blocksuite/blob/master/tests/selection.spec.ts#L1949

![image](https://user-images.githubusercontent.com/50035259/221397068-9a2340b5-1fce-4463-970a-f167b36600b6.png)